exchanges/binance/cfutures_types.go

```diff
@@ -206,30 +206,27 @@ type BatchCancelOrderData struct {
 
 // FuturesOrderPlaceData stores futures order data
 type FuturesOrderPlaceData struct {
-	ClientOrderID string  `json:"clientOrderID"`
+	OrderID       int64   `json:"orderId"`
+	Symbol        string  `json:"symbol"`
+	Pair          string  `json:"pair"`
+	Status        string  `json:"status"`
+	ClientOrderID string  `json:"clientOrderId"`
+	Price         float64 `json:"price,string"`
+	AvgPrice      float64 `json:"avgPrice,string"`
+	ExecuteQty    float64 `json:"executeQty,string"`
 	CumQty        float64 `json:"cumQty,string"`
 	CumBase       float64 `json:"cumBase,string"`
-	ExecuteQty    float64 `json:"executeQty,string"`
-	OrderID       int64   `json:"orderID,string"`
-	AvgPrice      float64 `json:"avgPrice,string"`
-	OrigQty       float64 `json:"origQty,string"`
-	Price         float64 `json:"price,string"`
+	TimeInForce   string  `json:"TimeInForce"`
+	OrderType     string  `json:"type"`
 	ReduceOnly    bool    `json:"reduceOnly"`
+	ClosePosition bool    `json:"closePosition"`
 	Side          string  `json:"side"`
 	PositionSide  string  `json:"positionSide"`
-	Status        string  `json:"status"`
-	StopPrice     int64   `json:"stopPrice"`
-	ClosePosition bool    `json:"closePosition"`
-	Symbol        string  `json:"symbol"`
-	Pair          string  `json:"pair"`
-	TimeInForce   string  `json:"TimeInForce"`
-	OrderType     string  `json:"type"`
-	OrigType      string  `json:"origType"`
-	ActivatePrice float64 `json:"activatePrice,string"`
```

Comment:
Why has this been removed?

Reply:
Reordered.

Reply:
Was your test order a trailing stop market order? `activatePrice` and `priceRate` are fields added for those order types. It seems like these are being removed unnecessarily

Reply:
Thanks for pointing that out! Fixed now.